### DISCUSSION TITLE
cant get Arctic Code Vault Contributor badge that i used have

### DISCUSSION THREAD

**Discussion by Light--** [community/community#49335] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

the badge just disappeared, last year i had it, i remember

**Comment by radar-blip** [comment]
@Light-- You still have it, its just hidden. Just click link below and unhide it.

https://github.com/Light--?tab=achievements

**Comment by YasinDehfuli** [comment]
there could be several reasons why it's no longer showing up. Here are a few things you can try to troubleshoot the issue:

1. Check your email: GitHub typically sends an email to contributors who have earned the Arctic Code Vault Contributor badge. If you still have access to the email address associated with your GitHub account, check to see if you received an email with information about the badge.

2. Check your profile settings: Make sure that the "Arctic Code Vault Contributor" badge is enabled in your profile settings. To do this, go to your GitHub profile, click on the "Edit Profile" button, and make sure that the "Arctic Code Vault Contributor" badge is selected under "Public contributions."

3. Check your contributions: The badge is based on your contributions to public repositories that were preserved in the Arctic Code Vault. Check to make sure that you've made significant contributions to a public repository that was included in the vault.

4. Reach out to GitHub support: If after trying the above steps you still can't see your Arctic Code Vault Contributor badge, you can try reaching out to GitHub support for assistance. They may be able to help you troubleshoot the issue and restore your badge if necessary.

I hope this helps you resolve the issue and earn your Arctic Code Vault Contributor badge again!
